Generals/Code/GameEngine/Source/GameClient/GUI/GUICallbacks/Menus/OptionsMenu.cpp

```diff
@@ -1228,8 +1228,8 @@ static void saveOptions( void )
 
 	//-------------------------------------------------------------------------------------------------
 	// Set System Time Font Size
-	val = TheWritableGlobalData->m_systemTimeFontSize; // TheSuperHackers @todo replace with options input when applicable
-	if (val)
+	val = pref->getSystemTimeFontSize(); // TheSuperHackers @todo replace with options input when applicable
+	if (val >= 0)
```

Comment:
This condition is always true, because `getSystemTimeFontSize` already clamps to 0.